helm/templates/config.yaml

```diff
@@ -12,6 +12,9 @@ data:
   BAG_API_CLIENT_MP_REST_URL: {{ required "Valid .Values.bagApi.url entry required!" .Values.bagApi.url }}
   BRON_ORGANISATIE_RSIN: {{ required "Valid .Values.organizations.bron.rsin entry required!" .Values.organizations.bron.rsin | quote }}
   BRP_API_CLIENT_MP_REST_URL: {{ required "Valid .Values.brpApi.url entry required!" .Values.brpApi.url }}
+  {{- if .Values.catalogusDomein }}
+  CATALOGUS_DOMEIN: {{ .Values.catalogusDomein }}
```

Comment:
Why do we need this new Helm Chart variable for this PR?

Reply:
Not directly related, but adds the ability to specify the domain in helm chart. Something found out during testing.

Reply:
ah I see!